SConstruct

```diff
@@ -19,56 +72,77 @@ elif sys.platform == 'win32':
 else:
     raise ValueError('Could not detect platform automatically, please specify with platform=<platform>')
 
-opts = Variables([], ARGUMENTS)
-
 opts.Add(EnumVariable('platform', 'Target platform', host_platform, ('linux', 'osx', 'windows')))
+opts.Add(PathVariable('osxcross_dir', 'Platform', '.'))
```

Comment:
Using the `OSXCROSS_ROOT` environment variable (if defined) instead of a SCons parameter would be better here, so that it's more consistent with Godot's build system.